docs/release-notes/3152-model-registry-cli.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Improvement**
+
+-  Revert changes to Model Registry CLI format so models can be referred to by a unique name, and
+   not only ID
```

Comment:
i would add a warning that between release x and y (i'm not sure the versions), if duplicate named models were added that they will be renamed.